ansible/init.sh

```diff
@@ -62,6 +62,15 @@ main() {
         execute_playbook $env $passwd_file
     done
 
+    # Grant the pipeline service account permissions to create projects
+    # -it cannot be done in Playbook, as ansible SA has no permissions to create
+    # cluster-scoped resources- and this command creates ClusterRoleBinding
+    for env in $environments;
+    do
+        oc adm policy add-cluster-role-to-user self-provisioner -z pipeline -n operator-pipeline-$env
```

Comment:
Can you add the oc command the the loop above? I don't see any benefit of duplicating a code.